src/rard/research/forms.py

```diff
@@ -876,7 +972,17 @@ class Meta:
             "date_range",
         )
 
+    bibliography_items = BibliographyModelMultipleChoiceField(
+        widget=forms.CheckboxSelectMultiple,
+        queryset=BibliographyItem.objects.all(),
+        required=False,
+    )
+
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        if self.instance and self.instance.is_anonymous_citing_author():
-            self.fields["order_name"].disabled = True
+        if self.instance:
+            self.fields[
+                "bibliography_items"
+            ].initial = self.instance.bibliography_items.all()
+            if self.instance.is_anonymous_citing_author():
+                self.fields["order_name"].disabled = True
```

Comment:
How come order_name is disabled for citing authors?

Reply:
Just for the anonymous citing author (it's a special instance who's order name is defined at the class level)